### PR TITLE
adds headpatting

### PR DESCRIPTION
**Pull request by imsxz**
## About The Pull Request
adds headpatting to the game

## Why It's Good For The Game
It will compliment the recent addition of the roleplaying server, as more diverse ways to interact with your coworkers make dynamic roleplaying situations more smooth.

## Changelog
:cl:imsxz
fix: you can now pat your coworkers on your head by targeting head and clicking them while on help intent.
/:cl:

### CODE REVIEW

**Comment by qbmax32** [comment]
heatpatting

**Comment by UrsoTheLobster** [comment]
headpat moment

**Comment by Twaticus** [comment]
im going to headpat imsxz irl for this PR

**Comment by imsxz** [comment]
how do i fix the title help

**Comment by imsxz** [comment]
@Twaticus bet

**Comment by cacogen** [comment]
*pats ur head*
that'll do pig, that'll do

**Comment by kriskog** [comment]
[](http://www.youtube.com/watch?v=YhImcEXca14 "")

**Comment by ATH1909** [comment]
Make this give a special moodlet for felinids, like the special felinid tabling moodlet.

**Comment by JJRcop** [comment]
see also
#21352
#24062
#24066
#24176
#28028
#33841

**Comment by PKPenguin321** [comment]
@kriskog playing the mindgame, posting that so that people will cringe hard enough to instinctively -1 this PR

**Comment by CRITAWAKETS** [comment]
no erp

**Comment by Borisvanmemes** [comment]
Fetish content, please speedmerge.

**Comment by cacogen** [comment]
On Hippie this was usually used as a condescending alternative to hugs. But
then Hippie is like a rough high school.

I dunno, this sucks due to the anime trash but I think when Pete added hugs
when people were complaining about the server turning into a hugbox it
probably prompted an overreaction when ultimately it did nothing. I don't
remember if anyone cared about borg pats.

**Comment by HomocidlePancakes** [comment]
Cringe

**Comment by trollbreeder** [comment]
Something something, allow nonhumans to equip pet collars again.

**Comment by Tlaltecuhtli** [comment]
does cacogen transmit messages from mars?
also no weeb shit thanks

**Comment by trollbreeder** [comment]
Tlalt i was thinking more of lizard slavery than catgirls on a leash.

**Comment by cacogen** [comment]
no it emails me and i respond to the emails. other people used to use it too but i guess they got scared

**Comment by Tharcoonvagh** [comment]
fetish content hello?????

**Comment by trollbreeder** [comment]
we need a clown emoji react

**Comment by Fikou** [comment]
image

**Comment by 81Denton** [comment]
hmmmm

**Comment by 81Denton** [comment]
hmmmm

**Comment by shellspeed1** [comment]
The replacement we need for cloning.

**Comment by trollbreeder** [comment]
denton is just too scared to speedmerge this masterpiece

**Comment by Tumesuo** [comment]
I hate m*intaners

**Comment by trollbreeder** [comment]
14 hours remain

**Comment by wesoda25** [comment]
You have brain damage

**Comment by wesoda25** [comment]
Also still not an emote so it won’t get merged anyways lol

**Comment by Iatots** [comment]
Don't you fucking jinx it wesoda.

**Comment by trollbreeder** [comment]
Oranges saves the day once again

**Comment by GravitronBlue** [comment]
>optimumtact commented on Aug 20, 2017
"there is a clique of people who use this emote and I don't like any of them and I was hoping they'd leave when it was removed"

This seems less like him saving the day again and more like:
>"You were the chosen one! It was said that you would destroy fetish content, not merge it.
You were to bring fun to TG, not lead it down the slippery slope!"

Anyway the old era of TG got outvoted and this is where our new senate Orangelord seems to be taking things. While personally against both this change and recent conversions to TG there's not much that can be done without some heavy opposition from within the coderbus itself.

I'll just be having to hope the [Feature] tag is just a joke in this continuing trend of headpat PRs.

**Comment by Iatots** [comment]
The clique became coderbus; it all started when cobby transitioned into anime avatars, that was the sign of a weeb coup de codage.

**Comment by imsxz** [comment]
headcoders are not called upon only to settle easy pull requests. These often settle themselves. It is where the balance quivers, and the proportions are veiled in mist, that the opportunity for codebase-saving decisions presents itself.

**Comment by wesoda25** [comment]
ok shadowflame

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -264,6 +264,9 @@
 			return
 		M.visible_message("<span class='notice'>[M] shakes [src] trying to get [p_them()] up!</span>", \
 						"<span class='notice'>You shake [src] trying to get [p_them()] up!</span>")
+	else if(check_zone(M.zone_selected) == "head") //Headpats!
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/carbon_defense.dm]:
```suggestion
	else if(check_zone(M.zone_selected) == BODY_ZONE_HEAD) //Headpats!

```

**Comment by imsxz** [comment]
https://prnt.sc/qwn8uc

**Comment by imsxz** [comment]
image

**Comment by 81Denton** [comment]
Just kidding... or am I?

**Comment by HomocidlePancakes** [comment]
24 hours have passed. Why is this still open?

**Comment by GuyonBroadway** [comment]
I've lost count of how many times this PR has been rejected...

**Comment by 81Denton** [comment]
1e8

**Comment by trollbreeder** [comment]
everyone press F for our lost cause

**Comment by wesoda25** [comment]
My guy

**Comment by kriskog** [comment]
Worst timeline

**Comment by imsxz** [comment]
where is the brigade of people that whine when PR's with less dislikes than likes are merged, when you need them?

**Comment by Twaticus** [comment]
this is an outrage

**Comment by ExcessiveUseOfCobblestone** [comment]
unless...?

**Comment by 81Denton** [comment]
Unless...?

**Comment by Twaticus** [comment]
re open this PR right now

**Pull request closed without merging**